docs/examples/800_istio.adoc

```diff
@@ -0,0 +1,169 @@
+
+tlsGateway.yaml
+[source,bash]
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: tlsgateway
+spec:
+  selector:
+    istio: ingressgateway # use istio default ingress gateway
+  servers:
+  - port:
+      number: 8043
+      name: tls
+      protocol: TLS
+    tls:
+      mode: SIMPLE
+      credentialName: "extend-credential" # must be the same as secret for the TLS key and cert
+      maxProtocolVersion: TLSV1_3
+    hosts:
+    - "*"
+----
+
+
+tlsVS.yaml
+[source,bash]
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: extend
+spec:
+  hosts:
+  - "*"
+  gateways:
+  - tlsgateway
+  tcp:
+  - match:
+    route:
+    - destination:
+        port:
+          number: 20000                    # the proxy server listening port
+        host: example-cluster-proxy-proxy  # the ClusterIP service name of the proxy server
+----
+
+Then a Coherence*Extend client can connect to the proxy server via TLS protocol at port 8043.
+
+== Using the Coherence operator with Istio
+
+NOTE: These instructions assume that you are using a Kubernetes cluster with Istio installed and configured already.
+
+You can deploy the operator into a namespace which has Istio automatic sidecar injection enabled. Before installing the operator, create the namespace in which you want to run the Coherence operator and label it.
+
+[source,bash]
+----
+kubectl create namespace coherence-operator
+----
+
+Istio Sidecar AutoInjection is done automatically if you label coherence-operator namespace with istio-injection. If you want to implement manual injection with the deployments then you need to use istioctl kube-inject.
+
+[source,bash]
+----
+kubectl label namespace coherence-operator istio-injection=enabled
+----
+
+After the namespace is labeled, you can install the operator using your preferred method in the Operator https://oracle.github.io/coherence-operator/docs/latest/#/installation/01_installation[Installation Guide].
+
+After installed operator, use the following command to confirm the operator is running:
+
+[source,bash]
+----
+kubectl get pods -n coherence-operator
+----
+
+== Creating a Coherence cluster with Istio support
+
+You can configure your cluster to run with Istio automatic sidecar injection enabled. Before creating your cluster, create the namespace in which you want to run the cluster and label it for automatic injection.
+
+[source,bash]
+----
+kubectl create namespace coherence-cluster
+kubectl label namespace coherence-cluster istio-injection=enabled
+----
+
+There is no other requirements to run Coherence in Istio environment.
+
+The following is an example that creates a cluster named example-cluster:
+
+example.yaml
+[source,bash]
+----
+# Example
+apiVersion: coherence.oracle.com/v1
+kind: Coherence
+metadata:
+  name: example-cluster
+----
+
+[source,bash]
+----
+$ kubectl -n coherence-cluster apply -f example.yaml
+----
+
```

Comment:
If using the Pod label works instead of labelling a namespace you might want to include an example of the Coherence yaml to add the required label to a Coherence deployment.
```
apiVersion: coherence.oracle.com/v1
kind: Coherence
metadata:
  name: storage
spec:
  labels:
    sidecar.istio.io/inject: "true"
```

Reply:
Tried this; it doesn't work.